Lib/subprocess.py

```diff
@@ -386,7 +386,8 @@ def _text_encoding():
 
 def call(*popenargs, timeout=None, **kwargs):
     """Run command with arguments.  Wait for command to complete or
-    timeout, then return the returncode attribute.
+    for timeout, specified in seconds, to expire. Then return the
+    returncode attribute.
```

Comment:
I think this simpler edit is sufficient.
```suggestion
    for timeout seconds, then return the returncode attribute.
```